Makefile

```diff
@@ -15,6 +15,20 @@
 #-----------------------------------------------------------------------------
 # Global Variables
 #-----------------------------------------------------------------------------
+
+# By default, Pilot, Mixer and CA will be built with debugger information.
```

Comment:
Let's change the default to 0 and maybe revert all changes to .circleci - there are probably other scripts and tools building istio - prow, cloud buildier.